css/index.css

```diff
@@ -43,31 +43,51 @@ h1:hover {
     cursor: default;
     color: pink;
 }
+
 /* hover each section*/
-h2:hover{
+h2:hover {
     cursor: default;
 }
-/* fix font for connect section*/
- #connect{
-    list-style: square;
-    display: block;
-    
+
+/* fix font for links in connect section*/
+#connect a {
+
+
+
+    padding: 15px;
     font-size: 30px;
     text-align: center;
 }
+
+/*styling the links for  a tags in connect section*/
+#connect a:hover {
+    border: 2px;
+    padding: 4px;
+    height: 100px;
+    width: 100px;
+    display: inline;
+
+    background: plum;
+}
+
 /*change color to black for any link*/
-a:any-link{
+a:any-link {
```

Comment:
oh wow, Today I Learned! I never knew of this `:any-link` pseudo-class, nice 🤩 😄